lib/concepts/lesson/operations/create_copy.rb

```diff
@@ -17,11 +17,29 @@ def call(lesson:, lesson_params:)
 
       private
 
-      # TODO: copy projects
       def build_copy(lesson, lesson_params)
-        copy = Lesson.new(parent: lesson, name: lesson.name, description: lesson.description)
-        copy.assign_attributes(lesson_params)
-        copy
+        lesson_copy = Lesson.new(parent: lesson, name: lesson.name, description: lesson.description)
+        lesson_copy.assign_attributes(lesson_params)
+
+        project_params = { name: lesson_copy.name, user_id: lesson_params[:user_id], lesson_id: lesson_copy.id }
+        lesson_copy.project = build_project_copy(lesson.project, project_params)
+
+        lesson_copy
+      end
+
+      def build_project_copy(project, project_params)
+        project_attributes = project.attributes.except('id', 'identifier').merge(project_params)
```

Comment:
should we be avoiding `created_at` and `updated_at` here and for the lesson? it seems like we can end up with some weird values, until updates occur otherwise

Reply:
Ah yes, thanks Dan, I've excluded these now